BaseFilter/sections/specific.txt

```diff
@@ -471,6 +471,7 @@ $subdocument,third-party,denyallow=disqus.com,domain=welovemanga.one
 ! NOTE: Specific rules
 !
 !
+lewdgames.to##div[id^="ai_widget"]
```

Comment:
As I see, it's a specific selector and I don't notice other similar elements.
So, you can add the domain here.
```adblock
afterschoolafrica.com,asianhobbyist.com###ai_widget-4
```